Makefile

```
@@ -249,6 +254,24 @@ dev-tor:  ## Run the development server with onion services in a Docker containe
 	@OFFSET_PORTS='false' DOCKER_BUILD_VERBOSE='true' USE_TOR='true' SLIM_BUILD=1 $(DEVSHELL) $(SDBIN)/run
 	@echo
 
+.PHONY:
+dev-get-id:  ## Get the ID of the running "make dev" or "make dev-tor" container.
+	@$(OCI_BIN) ps --format json --filter "name=securedrop-dev" --format '{{.ID}}'
+
+.PHONY:
+dev-enter:  ## Start a shell directly in the running "make dev" or "make dev-tor" container.
+	@$(OCI_BIN) exec -it \
+		$(shell make dev-get-id) \
```

This internal `make` command is outputting `make[1]: Entering directory /home/user/github/freedomofpress/securedrop ae8314c6d60a make[1]: Leaving directory /home/user/github/freedomofpress/securedrop`

so then obviously podman fails with: `Error: no container with name or ID "make[1]:" found: no such container`.

Solution appears to be:

```suggestion
		$(shell make -s dev-get-id) \
```

---

Fixed in c15991b48a04b516b62ad61450173968a8402aaf.